charts/images.yaml

```
@@ -293,9 +297,9 @@ images:
 
 # Logging
 - name: fluent-bit
-  sourceRepository: github.com/fluent/fluent-bit
-  repository: fluent/fluent-bit
-  tag: "1.9.7"
+  sourceRepository: https://github.com/fluent/fluent-operator
+  repository: kubesphere/fluent-bit
+  tag: "v2.0.6"
```

Nit

```suggestion
  tag: "v2.0.9"
```

I hope semver is respected and the upgrade to v2.0.9 is safe.